maxim/models/test_run.py

```
@@ -99,6 +121,18 @@ class YieldedOutputMeta:
     usage: Optional[YieldedOutputTokenUsage] = None
     cost: Optional[YieldedOutputCost] = None
 
+    def __init__(
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)